### DISCUSSION TITLE
非常炸裂的事情

### DISCUSSION THREAD

**Discussion by ghost** [Meloong-Git/PCL#3537] [讨论]
![Screenshot_2024-03-17-18-55-40-85_a252b927494330cdc2c8ba3b3f952e5e](https://github.com/Hex-Dragon/PCL2/assets/110439412/cff2a70b-8027-459b-8284-bc9a2a69b5b8)
![Screenshot_2024-03-17-18-55-05-02_a252b927494330cdc2c8ba3b3f952e5e](https://github.com/Hex-Dragon/PCL2/assets/110439412/91f7c160-11f4-44e0-b3b0-da2b0624814e)

大意是这个朋友想让linus把国家反诈中心加入到linux内核，提高全球网友的财产安全

https://github.com/torvalds/linux/pull/818

**Comment by allMagicNB** [comment]


**Reply by ghost** [reply]
可能是

**Comment by WorldHim** [comment]
虽然但是能不能别啥都发 GitHub 上，这里本身是要作为一个 bug 反馈平台，现在都要变成论坛灌水版了😶
这种几乎没啥关系的 q 群里发发就够了，没必要发 GitHub 上

**Reply by allMagicNB** [reply]
而且这和 PCL2 根本没有关联……

**Reply by wuliaodexiaoluo** [reply]
discussions 本身不就是灌水用的吗

**Reply by MoYuan-CN** [reply]
Discussion 区的定义是**讨论和 PCL 2有关的内容**或者**询问问题**用的

**Comment by Deep-Dark-Forest** [comment]
有没有人来close一下

**Reply by MoYuan-CN** [reply]
Closed
